### ISSUE TITLE
Can I passthrough a gpu through .conf file like usb_devices ?

### ISSUE THREAD

**Issue by mrroot912** [quickemu-project/quickemu#900]
Hello ..
Can I passthrough a gpu through .conf file like usb_devices ?

**Comment by lj3954** [comment]
No. There was discussion on this, but it hasn't gone anywhere as of yet. https://github.com/quickemu-project/quickemu/discussions/812

**Comment by shmahabi-dev** [comment]
This issue is being closed due to inactivity for 193 days. If this is still relevant, please comment or reopen and it will be re-opened. Thanks!

**Comment by shmahabi-dev** [comment]
This issue is being closed due to inactivity for 193 days. If this is still relevant, please comment or reopen and it will be re-opened. Thanks!
